audit-opentelemetry/plugindefinition.yaml

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: greenhouse.sap/v1alpha1
+kind: PluginDefinition
+metadata:
+    name: opentelemetry-audit
+spec:
+    version: 0.6.1
```

Comment:
Start with a new versioning, instead of following the original OTel plugin  versioning, the same is true for the helmChart.version